[2022-01-29 14:07:27,378] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: HomeWorkWeek2.get_data scheduled__2019-02-02T06:00:00+00:00 [queued]>
[2022-01-29 14:07:27,590] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: HomeWorkWeek2.get_data scheduled__2019-02-02T06:00:00+00:00 [queued]>
[2022-01-29 14:07:27,591] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-01-29 14:07:27,617] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-01-29 14:07:27,617] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-01-29 14:07:27,858] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): get_data> on 2019-02-02 06:00:00+00:00
[2022-01-29 14:07:27,875] {standard_task_runner.py:52} INFO - Started process 301 to run task
[2022-01-29 14:07:27,883] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'HomeWorkWeek2', 'get_data', 'scheduled__2019-02-02T06:00:00+00:00', '--job-id', '19', '--raw', '--subdir', 'DAGS_FOLDER/dag_questions1.py', '--cfg-path', '/tmp/tmpq4q0bbon', '--error-file', '/tmp/tmpi791fr3r']
[2022-01-29 14:07:27,884] {standard_task_runner.py:77} INFO - Job 19: Subtask get_data
[2022-01-29 14:07:28,199] {logging_mixin.py:109} INFO - Running <TaskInstance: HomeWorkWeek2.get_data scheduled__2019-02-02T06:00:00+00:00 [running]> on host d1e1fe005577
[2022-01-29 14:07:29,104] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=HomeWorkWeek2
AIRFLOW_CTX_TASK_ID=get_data
AIRFLOW_CTX_EXECUTION_DATE=2019-02-02T06:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-02-02T06:00:00+00:00
[2022-01-29 14:07:29,106] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-01-29 14:07:29,140] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo "helloworld!!"']
[2022-01-29 14:07:29,189] {subprocess.py:85} INFO - Output:
[2022-01-29 14:07:29,195] {subprocess.py:89} INFO - helloworld!!
[2022-01-29 14:07:29,196] {subprocess.py:93} INFO - Command exited with return code 0
[2022-01-29 14:07:29,380] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=HomeWorkWeek2, task_id=get_data, execution_date=20190202T060000, start_date=20220129T140727, end_date=20220129T140729
[2022-01-29 14:07:29,568] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-01-29 14:07:30,164] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
